datamodel/12_2_roles.sql

```diff
@@ -0,0 +1,106 @@
+-------------------------------------------
+/* CREATE roles - cluster initialisation */
+-------------------------------------------
+DO $$
+DECLARE
+    role text;
+BEGIN
+    FOREACH role IN ARRAY ARRAY['tww_viewer', 'tww_user', 'tww_manager', 'tww_sysadmin'] LOOP
```

Comment:
This code is duplicated with 12_0_roles.sql

Reply:
resolved by pythonising